### DISCUSSION TITLE
Challenge #4: Developing Post-Disaster Housing Response

### DISCUSSION THREAD

**Discussion by jacobrharris** [datakind/datakit-housing-fall-2024#5] [Challenges]
# Developing Post-Disaster Housing Response

In the wake of a natural disaster, recovery organizations and local governments often lack the right level of insights to inform which properties should be targeted for disaster relief support to ensure that affordable housing remains available in communities post-disaster. In many situations, post-disaster task forces use anecdotal information to form an understanding of the damage to affordable housing. Further, decision-makers often don’t have the data they need to understand where impacted housing units are located and who lives in these affected units. Without this data, proactive assignment of resources is limited, and affordable housing units are often lost to private real estate development organizations who might engage with property owners on a faster timeline, forever changing the makeup of communities. Research [suggests](https://today.tamu.edu/2022/02/10/disasters-can-wipe-out-affordable-housing-forever-unless-communities-plan-ahead-that-loss-hurts-the-economy/#:~:text=Research%20suggests%20that%20affordable%20housing,return%20to%20their%20original%20development) that affordable housing will almost always be replaced by more expensive housing targeted to a wealthier demographic. [According to NOAA](https://www.climate.gov/news-features/blogs/beyond-data/2022-us-billion-dollar-weather-and-climate-disasters-historical#:~:text=2022%20Highlights,the%2020%20events%20in%202021.) the three most recent years on record (2020, 2021, and 2022\) are the three worst years on record for the number of separate weather and climate disasters that cause $1B+ of damage.

The big question a housing stakeholder is hoping to answer with results of this challenge is: how can we proactively plan for disaster response, and when disaster hits, where can we direct efforts to the places where they will have an outsized impact by assisting the most marginalized in society: low-income households, migrants, and communities of color?

The big question a resident is hoping to answer with the results of this challenge is: what is the disaster risk in my community, and how can I get information and seek support if a disaster happens?

## Community focus: Florida

Bright Community Trust (Bright) is an organization that works across the state to support economic opportunity through strengthening community resources and access to homeownership. As a partner to many local governments, Bright is a leader in post-disaster response and rebuild efforts. Bright hopes to understand where to recommend directing post-disaster recovery efforts and funding in the months and years post-disaster, and to be able to share those insights sooner with key collaborators and policymakers. Bright would like to understand the potential risk landscape for the regions they serve in the southwest (Fort Myers), suncoast (Tampa, St. Petersburg, Port Richey), and northwest Florida (Port St. Joe) areas so that they can collaborate to implement economic support plans prior to disaster.

## Get started with existing data

[Grab the data here](https://github.com/datakind/datakit-housing-fall-2024/releases/latest).

Create an understanding of community vulnerability to disasters using data from the EODatascape such as the FEMA (2014-2021) \- Expected building loss rate (Natural Hazards Risk Index), FEMA (2014-2021) \- Expected population loss rate (Natural Hazards Risk Index), DOE (2018) \- Energy burden (percentile). At a Census tract level, this information can help create an understanding of vulnerability and potential loss. Consider exploring population demographic and economic information alongside this disaster information to start to shape an understanding of where vulnerable communities are located and characteristics of those communities. 

This “getting started” analysis should help answer the following questions

* Where are communities located that have higher vulnerability to natural disasters?  
* Who is represented in those communities?  
* What is the housing make-up of those communities?  
* Analysis insights and questions: What surprised you from this analysis? What are some limitations of the analysis? What are ways to extend the work?

## Take it further

Research: The EODatascape data just scratches the surface of information regarding potential vulnerability to natural disasters. Research additional data sets, case studies, and innovations that have helped communities prepare for disaster response and resilience.   
Data analysis and visualization: Using additional datasets, dig deeper into information regarding disaster vulnerability and consider how that information might be used for proactive community support. Look historically at disasters in case study states of Florida and California and generate exploration and insights regarding the impact of disasters and what indicators might have been in place for proactive support.   
Prototyping:

* What might the shape of an early warning system look like for a disaster situation? How might individuals receive accurate and timely information if they are located in vulnerable areas? What type of warnings might need to be issued for disasters that don’t rise to a “state of emergency”?  
* How might you prototype a tool for housing authorities and developers to assess climate risks and inform housing development strategies?   
* What are the most effective community engagement strategies to ensure vulnerable communities benefit from climate adaptation investments?

## Additional data sets to explore

* [https://www.census.gov/programs-surveys/community-resilience-estimates.html](https://www.census.gov/programs-surveys/community-resilience-estimates.html)  
* [https://screeningtool.geoplatform.gov/en/methodology](https://screeningtool.geoplatform.gov/en/methodology)  
* [https://resilience.climate.gov/datasets/FEMA::national-risk-index-census-tracts/about](https://resilience.climate.gov/datasets/FEMA::national-risk-index-census-tracts/about)  
* [https://www.census.gov/topics/preparedness.html](https://www.census.gov/topics/preparedness.html)  
* [https://www.napsgfoundation.org/a-census-bureau-data-tool-for-assessing-disaster-impacts/](https://www.napsgfoundation.org/a-census-bureau-data-tool-for-assessing-disaster-impacts/)  
* [https://www.fema.gov/partnerships/community-disaster-resilience-zones](https://www.fema.gov/partnerships/community-disaster-resilience-zones)  
* [https://www.fema.gov/data-visualization/disaster-housing-assistance](https://www.fema.gov/data-visualization/disaster-housing-assistance)

**Comment by racehd** [comment]
I've added some basic visualizations for vulnerability #25, though for now this is focused on individual attributes while I familiarized myself with the data and geospatial visualizations
